File: src/main/java/io/xol/chunkstories/api/exceptions/content/mods/ConflictingExternalCodeException.java
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package io.xol.chunkstories.api.exceptions.content.mods;

import io.xol.chunkstories.api.content.mods.Mod;

public class ConflictingExternalCodeException extends ModLoadFailureException
{

	/**
	 * 
	 */
	private static final long serialVersionUID = -6933179574010715068L;

	public ConflictingExternalCodeException(Mod mod, String message)
	{
		super(mod, message);
	}

}

File: src/main/java/xyz/chunkstories/api/exceptions/content/mods/ConflictingExternalCodeException.kt
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.exceptions.content.mods

import xyz.chunkstories.api.content.mods.Mod

class ConflictingExternalCodeException(mod: Mod, message: String) : ModLoadFailureException(mod, message) {
    companion object {

        /**
         *
         */
        private val serialVersionUID = -6933179574010715068L
    }

}
